scripts/update/Updater.php

```diff
@@ -389,5 +392,28 @@ public function update($initialVersion){
 
             $this->setVersion('16.0.0');
         }
+
+        if ($this->isVersion('16.0.0')) {
+            $importerConfig = [
+                MetadataImporter::INJECTOR_KEY => ['oat\\taoQtiItem\\model\\qti\\metadata\\ontology\\LomInjector'],
```

Comment:
In order to make these lines look a bit cleaner, we can use the `LomInjector::class` notation instead.